### PR TITLE
Activity Events Frontend

### PR DESCRIPTION
**Pull request by minhngo3818**
Design gui for event page

- [x]  #22 
- [x] #23

### CODE REVIEW

**Comment by minhngo3818** [comment]
Re-design event box, add animations for buttons and expansion and add tooltip

**Comment by minhngo3818** [comment]
Mobile responsive feature

**Comment by KHVBui** [comment]
I'll review and push this through tomorrow morning!

File: src/activity/events/events.css [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+		margin: 10px auto;
+		font-size: 24px;
+		border: 3px solid #ffa611;
+	}
+	.event {
+		display: flex;
+		flex-direction: column;
+	}
+
+	.event-date {
```
Comment (KHVBui) [inline src/activity/events/events.css:246]: Not sure if I like how far apart the month and day are when it's adapted to a smaller screen, but that's something we can talk about later.

**Review by KHVBui — APPROVED** [review]

**Comment by KHVBui** [comment]
Note, the "Activity" button on the navigation bar still leads to this default screen with the old "Events" visuals. We might want to think about removing the ability to press on "Activity" so people don't accidentally land here and miss the new "Events" page.
Screen Shot 2022-08-12 at 12 31 44 PM

**Comment by minhngo3818** [comment]
I'll push a commit for disabling activity page since it's not needed so far. I intend to make something fancy on that page.

File: src/activity/events/events.css (hunk 1 above)
Comment (minhngo3818) [reply src/activity/events/events.css:246]: I'll figure the date layout for responsive mod before merging

**Pull request merged**